systers_portal/community/views.py

```diff
@@ -8,6 +11,11 @@
 from community.mixins import CommunityMenuMixin
 from community.models import Community
 from common.mixins import UserDetailsMixin
+from community.models import CommunityPage
```

Comment:
You can import the `CommunityPage` on the same line with `from community.models import Community`.